dev/com.ibm.ws.classloading/resources/com/ibm/ws/classloading/internal/resources/ClassLoadingServiceMessages.nlsprops

```diff
@@ -155,3 +155,7 @@ cls.classloader.config.typo.useraction=Check the server configuration for invali
 cls.classloader.config.typo2=CWWKL0083E: When using prefix + or -, [{0}] API type is not valid in this [{1}] list.  All API type must have a prefix of + or -
 cls.classloader.config.typo2.explanation=Change the invalid API type to a valid API type with + or -.
 cls.classloader.config.typo2.useraction=Check the server configuration for invalid API type to ensure that the API type are correct.
+
+cls.classloader.suggested.feature=CWWKL0084W: Unable to load class {0}, try enabling the {1} (or newer) feature in your server configuration.
+cls.classloader.suggested.feature.explanation=A class could not be loaded which is typically provided by one or more server features.
```

Comment:
Change "A class could not be loaded which is typically provided by one or more server features." to "A class that is typically provided by one or more server features could not be loaded.".